src/context/fetch.ts

```diff
@@ -26,20 +26,25 @@ export const augmentRequestInit = (requestInit: RequestInit): RequestInit => {
   }
 }
 
+export const gracefullyFetch = <ResponseType = any>(
```

Comment:
What do you think if we drop this graceful handling entirely? This encapsulated body resolution behavior doesn't help much, and using explicit `.text`, `.json`, etc. is much better for both compliance with an actual fetch and versatility.

Reply:
yes we can, is something that can be dropped